src/main/scala/FriendsByAge.scala~

```diff
@@ -0,0 +1,53 @@
+import org.apache.spark._
```

Comment:
File extension scala~ is incorrect
import org.apache.log4j._ (do not use underscore use specific object in import statements)